### PR TITLE
added tests for search experience on search page

### PR DESCRIPTION
**Pull request by pbadak**

### CODE REVIEW

File: suites/test_search_beta.py [hunk 1]
```diff
@@ -55,9 +56,40 @@ def search_for_repo(self):
...
+                    By.CSS_SELECTOR, locators.FIRST_MODULE_LISTED_CSS), contains_string(constants.published_module))
+        check_that("Title is displayed on search page", utilities.find_element(self.driver, By.XPATH,
+                           locators.TITLES_ON_SEARCH_PAGE_XPATH).is_displayed(), is_true())
+        check_that("Repository is displayed on search page", utilities.find_element(self.driver, By.XPATH,
+                           locators.REPOSITORY_ON_SEARCH_PAGE_XPATH).is_displayed(), is_true())
+        check_that("Upload date is displayed on search page", utilities.find_element(self.driver, By.XPATH,
+                           locators.UPLOAD_DATE_ON_SEARCH_PAGE_XPATH).is_displayed(), is_true())
+        check_that("Last Published Date is displayed on search page", utilities.find_element(self.driver, By.XPATH,
+                           locators.LAST_PUBLISHED_DATE_ON_SEARCH_PAGE_XPATH).is_displayed(), is_true())
+        check_that("pagination is enabled on search page", utilities.find_element(self.driver, By.XPATH,
```
Comment (noopurmath3) [inline suites/test_search_beta.py:76]: This test will surely fail as we are filtering the titles which returns < 5 titles hence, pagination will always be disabled

**Review by noopurmath3 — CHANGES_REQUESTED** [review]
Pushed some changes to this branch and added a comment as well.

File: suites/test_search_beta.py (hunk 1 above)
Comment (pbadak) [reply suites/test_search_beta.py]: changes has been pushed .. thanks for your review :)

**Comment by pbadak** [comment]
merging this PR as discussed @noopurmath3

**Pull request merged**